src/View/View.php

```diff
@@ -1531,7 +1561,12 @@ protected function _paths(?string $plugin = null, bool $cached = true): array
                 return $this->_pathsForPlugin[$plugin];
             }
         }
-        $templatePaths = App::path(static::NAME_TEMPLATE);
+
+        $prefixPath = $this->prefixPath ? $this->prefixPath . DIRECTORY_SEPARATOR : '';
```

Comment:
We have a `subDir` already which is inserted in a similar position in the template path. Would that potentially work for bake as well?

Reply:
So, `subdir` only affects template paths and is inserted after the `templatePath`. It does not affect element and layout paths.